### PR TITLE
RegistryEvent - Rearrangement

### PR DESCRIPTION
**Pull request by yaronMSFT**
- Rearranging files
- Generating uniform readme files

### CODE REVIEW

**Review by oshezaf — CHANGES_REQUESTED** [review]
- You did not rename the yaml files to the function alias
- I think you missed on this one: "Update titles and descriptions to latest standard (see web)"
- I did not see product name changes. Did you check that they are per the [doc](https://docs.microsoft.com/en-us/azure/sentinel/normalization-about-schemas#vendors-and-products)?
- Also: Make sure you update the templates using these parsers (AsimFullDeployed, Sysmon, Windows)  to use the new parameters when calling these templates.

File: Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSecurityEvent.yaml [hunk 1]
```diff
@@ -12,7 +12,8 @@ References:
   Link: https://aka.ms/ASimRegistryEventDoc
 - Title: ASIM
   Link: https:/aka.ms/AboutASIM
-Description: ASIM Security Events Registry Creation Event Parser (event number 4657)
+Description: |
+  This ASIM parser supports normalizing Microsoft Security Event logs, produced by the Microsoft AMA connector, to the ASIM Registry Event normalized schema (event number 4657).
```
Comment (oshezaf) [inline Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSecurityEvent.yaml:16]: -> "This ASIM parser supports normalizing Microsoft Security Events (event number 4657), produced by the Microsoft AMA connector, to the ASIM Registry Event normalized schema."

Changes (which will repeat below):
- Moved (...) earlier.
- Event logs -> Events
- Dot at the end

File: Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSysmon.yaml [hunk 2]
```diff
@@ -12,7 +12,8 @@ References:
   Link: https://aka.ms/ASimRegistryEventDoc
 - Title: ASIM
   Link: https:/aka.ms/AboutASIM
-  Description: ASIM Sysmon Registry Creation Event Parser (event number 12, 13, 14) from "Event" and "WindowsEvent" tables.
+Description: |
+      This ASIM parser supports normalizing Microsoft Sysmon logs, produced by the Windows Forwarded Events connector, to the ASIM Registry Event normalized schema (event number 12, 13, 14) from "Event" and "WindowsEvent" tables.
```
Comment (oshezaf) [inline Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSysmon.yaml:16]: -> "This ASIM parser supports normalizing Microsoft Sysmon events (event number 12, 13, 14) from "Event" and "WindowsEvent" tables, to the ASIM Registry Event normalized schema .

I removed the connector name as Event is produced by another (yet to be identified) connector.

File: Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Parser:
-  Title: ASIM windows Event Registry Creation Event Parser
+  Title: egistry Event ASIM parser for Microsoft Windows Event (registry creation event)
```
Comment (oshezaf) [inline Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml:2]: Missing R at the beginning, "Windows Event" -> "Windows Events"

File: Parsers/ASimRegistryEvent/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+For more information, see:
+
+- [Normalization and the Advanced Security Information Model (ASIM)](https://aka.ms/AboutASIM)
+- [Deploy all of ASIM](https://aka.ms/DeployASIM)
+- [ASIM RegistryEvent normalization schema reference](https://aka.ms/ASimRegistryEventDoc)
+
+<br>
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/ASimRegistryEventARM) [![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://aka.ms/ASimRegistryEventARMgov)
```
Comment (oshezaf) [inline Parsers/ASimRegistryEvent/README.md:15]: The gov AKA link is missing.

File: Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml [hunk 5]
```diff
@@ -12,7 +12,8 @@ References:
   Link: https://aka.ms/ASimRegistryEventDoc
 - Title: ASIM
   Link: https:/aka.ms/AboutASIM
-Description: ASIM Windows Event Registry Creation Event Parser (event number 4657)
+Description: |
+        This ASIM parser supports normalizing Microsoft Windows Event logs, produced by the Windows Forwarded Events connector, to the ASIM Registry Event normalized schema (event number 4657).
```
Comment (oshezaf) [inline Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml:16]: -> "This ASIM parser supports normalizing Microsoft Windows events (event number 4657), produced by the Windows Forwarded Events connector, to the ASIM Registry Event normalized schema."


"

File: Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml [hunk 6]
```diff
@@ -12,7 +12,8 @@ References:
...
 ParserName: vimRegistryEventMicrosoftWindowsEvent
 ParserQuery: | 
               let RegistryType = datatable (TypeCode:string,TypeName:string)
```
Comment (oshezaf) [inline Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml:19]: EventProduct should 'Security Events'

**Review by oshezaf — CHANGES_REQUESTED** [review]

File: Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSecurityEvent.yaml [hunk 7]
```diff
@@ -1,9 +1,9 @@
 Parser:
-  Title: ASIM Security Events Registry Creation Event Parser
+  Title: Registry Event ASIM parser for Microsoft Security Event
   Version: '0.1'
   LastUpdated: June 23, 2021
 Product:
-  Name: Microsoft Windows Security Events
+  Name: Microsoft Security Event
```
[the remark this answers is not in the document]
Comment (yaronMSFT) [reply Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSecurityEvent.yaml:6]: The table is Security Event. Are you sure about that?

File: Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml (hunk 6 above)
Comment (YaronFruchtmann) [reply Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml:19]: What is the rule we establish here? maybe the product for WEF should be the (etw) provider?

Comment (oshezaf) [reply Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftWindowsEvent.yaml:19]: We decided that the "Type" field refers to the table the data is stored in, while Product and Vendor reflect the true source.

File: Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSecurityEvent.yaml (hunk 7 above)
[the remark this answers is not in the document]
Comment (oshezaf) [reply Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSecurityEvent.yaml:6]: Yes

**Review by oshezaf — CHANGES_REQUESTED** [review]
Left two responses which I believe still require changing + the aka.ms gov link.

File: Parsers/ASimRegistryEvent/README.md (hunk 4 above)
Comment (yaronMSFT) [reply Parsers/ASimRegistryEvent/README.md:15]: missing from aka! ok...

**Review by oshezaf — APPROVED** [review]

**Pull request merged**